### ISSUE TITLE
new bug after latest bugfix

### ISSUE THREAD

**Issue by joujoucoons** [DIGITALCRIMINALS/OnlyFans#541]
after latest bugfix when choosing a model it will always download every model and ignore the choice made

**Comment by Neurosis404** [comment]
same here. But for me, it isn't downloading anything. Script is just running through and doing nothing

**Comment by josephlopez86** [comment]
how do you download the bugfix? I've only been downloading from the releases secton

**Comment by Neurosis404** [comment]
> how do you download the bugfix? I've only been downloading from the releases secton

I had a strange error this morning (something with str and get, can't remember exactly), so I decided to run the updater.py

**Comment by Rhysdabeast** [comment]
> how do you download the bugfix? I've only been downloading from the releases secton

https://github.com/DIGITALCRIMINALS/OnlyFans/archive/refs/heads/master.zip

**Comment by MonkeyKingViper** [comment]
> how do you download the bugfix? I've only been downloading from the releases secton

Go to the commits and find  c15051a and download it. Or update the line of code being edited if you have 85aec02 like I did.

**Comment by MonkeyKingViper** [comment]
> same here. But for me, it isn't downloading anything. Script is just running through and doing nothing

Yeah my stuff was working a couple days ago but I just bought a PPV from someone and it's not downloading. I even deleted all the metadata to make sure it would see it as a new item. I wonder how much is being missed right now.

**Comment by ElSandu** [comment]
Hi guys.

Any idea on how to apply the bugfix in the docker version?
